src/QMCDrivers/QMCDriverNew.h

```diff
@@ -85,6 +85,32 @@ class QMCDriverNew : public QMCDriverInterface, public MPIObjectBase
    */
   std::bitset<QMC_MODE_MAX> qmc_driver_mode_;
 
+  /** The timers for the driver.
+   *
+   * This cleans up the driver constructor, and a reference to this structure 
+   * Takes the timers into thread scope.
+   */
+  struct Timers
```

Comment:
Is DriverTimers a better name?

Reply:
Thinking about this even though its a nested type its less readable used as a argument to the threaded functions.  I'll change to `DriverTimers`

Reply:
Done